apps/settings/locales/device_type/phone/settings.device.en-US.properties

```diff
@@ -12,9 +12,12 @@ findmydevice-active-tracking = Your phone is being tracked by another device thr
 findmydevice-not-tracking = No devices are actively tracking your phone through the website.
 ums-confirm = Enabling USB storage allows another computer or device connected using USB to access files on this phone.
 reset-device = Reset Phone
-reset-warning-body = Resetting will erase all your data, including any apps you’ve purchased. The phone will be restored to its factory condition. None of your settings or data will be saved.
+reset-warning-body = Resetting will erase all your data.<br> Including any apps you’ve purchased. The phone will be restored to its factory condition.<br> None of your settings or data will be saved.
 reset-devtools-warning-body = Resetting will erase all your data, including any apps you’ve purchased. None of your settings or data will be saved. But your phone will have unrestricted developer tools, allowing you to modify and debug system applications.
 powerSave-explanation = Turning on power save mode turns off the phone’s data, Bluetooth and Geolocation connections to extend battery life. You can still turn these services back on manually.
 performanceAndUsageDataInfo = Help us improve {{brandShortName}} by sharing data about your phone and the way you use it.
+# Add new with Help to improve firefox OS
+helpToImproveFirefoxOS =Help to improve Firefox OS
+helpToImproveFirefoxOSMessage=Help us improve Firefox OS by sharing data about your phone and the way you use it.
```

Comment:
Note the {{brandShortName}} just above these strings. Use that instead of saying "Firefox OS". That's how we deal with releases where the trademark is not licensed.